### PR TITLE
Feat / apply autocomplete attribute to input fields

### PR DESCRIPTION
**Pull request by MelissaDTH**
## This PR

- Applies `autocomplete` to all relevant input fields, for reference see: [HTML attribute: autocomplete
](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)
   - Solves ticket: OTT-384

### CODE REVIEW

**Review by langemike — APPROVED** [review]
LGTM! You only need to update the snapshots ;)

File: packages/ui-react/src/components/EditPasswordForm/EditPasswordForm.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const EditPasswordForm: React.FC<Props> = ({
           required
         />
       )}
-      {showResetTokenField && (
+      {true && (
```
Comment (ChristiaanScheermeijer) [inline packages/ui-react/src/components/EditPasswordForm/EditPasswordForm.tsx]: I think you did this for testing? 😅

**Review by ChristiaanScheermeijer — COMMENTED** [review]
Very nice! Only snapshots (which Mike already mentioned) and a testing condition needs to be fixed

File: packages/ui-react/src/components/EditPasswordForm/EditPasswordForm.tsx (hunk 1 above)
Comment (MelissaDTH) [reply packages/ui-react/src/components/EditPasswordForm/EditPasswordForm.tsx]: Oops! That's correct, thanks for this!

**Review by ChristiaanScheermeijer — APPROVED** [review]

**Pull request merged**